docs/internals/contributing/writing-code/pull-requests.txt

```diff
@@ -327,8 +333,6 @@ benchmarks can be run on a pull request by labeling the pull request with
 
 .. _django-asv: https://github.com/django/django-asv/
 
-.. _patch-review-checklist:
-
```

Comment:
I'd put this back

Reply:
Alright, I can live with that compromise 😁